.github/workflows/check.yml

```diff
@@ -0,0 +1,28 @@
+name: CI Check
+
+on: [pull_request]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Checkout Extension SDK
+        run:
+          git clone https://github.com/hivemq/hivemq-extension-sdk.git ../hivemq-extension-sdk ||
+          cd ../hivemq-extension-sdk ||
+          git checkout ${GITHUB_REF##*/} || true ||
+          cd ../hivemq-community-edition
```

Comment:
maybe add a multiline command instead of the pipes (https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun)